docs/src/pages/Landing.vue

```diff
@@ -1,71 +1,63 @@
-          q-btn(color="teal", no-caps, no-wrap, :icon-right="mdiLaunch", label="PWA (Progressive Web App)", to="/quasar-cli/developing-pwa/introduction")
-          q-btn(color="pink", no-caps, no-wrap, :icon-right="mdiLaunch", label="BEX (Browser Extension)", to="/quasar-cli/developing-browser-extensions/introduction")
-          q-btn(color="accent", no-caps, no-wrap, :icon-right="mdiLaunch", label="SSR (Server-side Rendered App)", to="/quasar-cli/developing-ssr/introduction")
-          q-btn(color="orange", no-caps, no-wrap, :icon-right="mdiLaunch", label="Hybrid Mobile App", to="/quasar-cli/developing-mobile-apps")
-          q-btn(color="indigo", no-caps, no-wrap, :icon-right="mdiLaunch", label="Multi-platform Desktop App", to="/quasar-cli/developing-electron-apps/introduction")
-
-  section.bg-white.text-grey-10.text-center.row.justify-center
-    .landing__features.row
-      .col-12.q-my-xl.row.items-center.justify-center
-        q-img.col-12.col-sm-4.landing__icongenie-logo(src="https://cdn.quasar.dev/img/iconfactory.png")
-        .col-12.col-sm-8.column.items-center
-          h4 Quasar Icon Genie
-          .q-mt-lg.q-mb-xl Consumes a source icon and automatically clones, scales, minifies and places the (needed) 100+ generated icons and splash screens in the appropriate directories for you.
-          q-btn(color="brand-primary", unelevated, no-caps, no-wrap, :icon-right="mdiLaunch", label="IconGenie", to="/icongenie/introduction")
+<template>
+  <q-page class="landing column justify-center bg-lp-light-grey q-px-xl font-monserrat">
+    <h1 class="lp-heading lp-heading--large landing__title normal-line-height">Welcome to quasar docs</h1>
+    <p
+      class="letter-spacing-300 text-size-16 text-weight-bold text-center text-lp-deep-dark"
+    >Here you can find everything you need to start working with Quasar.</p>
+    <p
+      class="letter-spacing-40 text-size-16 text-lp-dark text-center"
+    >In the top navigation bar there is a search function that helps you find what you need and also
+      <br v-if="$q.screen.gt.sm"> many other pages that you can explore, like beginner resources and other cool content
+      about <br v-if="$q.screen.gt.sm"> the Quasar galaxy. Don’t forget to check our page to
+      <a
+        class="text-lp-accent"
+        href="https://github.com/sponsors/rstoenescu"
+        target="_blank">become sponsor</a>!
+    </p>
+    <p class="letter-spacing-40 text-size-16 text-lp-dark text-center">If you are an experienced explorer get right into
+      action: use the navigation drawer on the left <br v-if="$q.screen.gt.sm"> to navigate through our most important
+      technical resources.
+    </p>
+    <h2
+      class="lp-heading lp-heading--medium q-mb-lg cards-title normal-line-height"
+    >Have a look at Our most used pages</h2>
+    <div class="cards-grid lp-mb--large">
```

Comment:
I'm pretty sure the whole card must be clickable here, not just the button, but ask Nick to be sure